sdk/eventhub/event-hubs/README.md

```diff
@@ -1,259 +1,198 @@
+  // NOTE: For receiving events from Azure Stream Analytics, please send Events to an EventHub where the body is a JSON object.
+  // const eventData = { body: { "message": "Hello World" }};
+  await client.send({
+  body: "my-event-body"
+}, "partitionId");
+```
 
-async function main() {
-  const onError = (err) => {
-    console.log("An error occurred on the receiver ", err);
-  };
-  
-  const onMessage = (eventData) => {
-    console.log(eventData.body);
-    const enqueuedTime = eventData.annotations["x-opt-enqueued-time"];
-    console.log("Enqueued Time: ", enqueuedTime);
-  };
+### Receive events
 
-  const receiveHandler = client.receive("1", onMessage, onError, { eventPosition: EventPosition.fromEnqueuedTime(Date.now()) });
+Once you have created an instance of an `EventHubClient` class, you can receive events in one of 2 ways:
 
-  // To stop receiving events later on...
-  await receiveHandler.stop();
-}
+   - [Get an array of events](#get-an-array-of-events)
+   - [Register event handler](#register-event-handler)
 
-main().catch((err) => {
-  console.log(err);
-});
-```
+#### Get an array of events
 
-## Example 2.2 - Receive specified number of events for a given time
-This mechanism is useful when you want to see how the received events look like. It can also be useful for debugging purpose.
+Use the [receiveBatch](https://docs.microsoft.com/en-us/javascript/api/@azure/event-hubs/eventhubclient?view=azure-node-latest#sendbatch-eventdata----string---number-) function which returns a promise that resolves to an array of events.
 
-Receive events from partitionId `"1"` after the current time.
-```js
-const { EventHubClient, EventPosition } = require('@azure/event-hubs');
+```javascript
+const myEvents = await client.receiveBatch("partitionId", 10);
+```
 
-const client = EventHubClient.createFromConnectionString(process.env["EVENTHUB_CONNECTION_STRING"], process.env["EVENTHUB_NAME"]);
+#### Register event handler
 
-async function main() {
-  const datas = await client.receiveBatch("1", 100 /*number of events*/, 20 /*amount of time in seconds the receiver should run. Default 60 seconds.*/, { eventPosition: EventPosition.fromEnqueuedTime(Date.now()) });
-  console.log("Array of EventData objects", datas);
-}
+Use the [receive](https://docs.microsoft.com/en-us/javascript/api/@azure/event-hubs/eventhubclient?view=azure-node-latest#receive-string---number--onmessage--onerror--receiveoptions-) to set up event handlers and have it running as long as you
+need. When you are done, call `receiveHandler.close()` to stop receiving any more events.
 
-main().catch((err) => {
-  console.log(err);
-});
+```javascript
+const myEventHandler = async (event) => {
+  // your code here
+};
+const myErrorHandler = (error) => {
+  console.log(error);
+};
+client.receive("partitionId", myEventHandler, myErrorHandler);
 ```
 
-## Example 3 - Send an event with partition key.
+### Get an event hub runtime info
```

Comment:
We need to add only the main scenarios as samples in the readMe. i don't think `getHubRuntimeInformation` and `getPartitionInformation` qualify here. See the template for reference